components/rich_text_editor/extensions/suggestion/SuggestionList.vue

```diff
@@ -89,10 +91,26 @@ export default {
 
     upHandler () {
       this.selectedIndex = ((this.selectedIndex + this.items.length) - 1) % this.items.length;
+
+      const activeElement = this.$refs.suggestionList.querySelector('.dt-list-item--highlighted');
+      if (activeElement) {
+        activeElement.scrollIntoView({
+          behaviour: 'smooth',
+          block: 'center',
+        });
+      }
```

Comment:
Instead of repeating in both functions, isolate in a single function `scrollActiveElementIntoView` or somethin like that.